tools/cxbe/Main.cpp

```diff
@@ -54,6 +69,87 @@ int main(int argc, char *argv[])
         szXbeTitle[40] = '\0';
     }
 
+    // interpret title id
+    if(szXbeTitleID[0])
+    {
+        bool hex = true;
+        for(int i = 0; szXbeTitleID[i]; ++i)
+        {
+            if(szXbeTitleID[i] == '-')
+            {
+                hex = false;
+                break;
+            }
+        }
+        if(hex)
+        {
+            sscanf(szXbeTitleID, "%x", &dwTitleId);
+        }
+        else
+        {
+            char titlechar[2] = { 'C', 'X' };
+            unsigned titleno = 9999;
+            sscanf(szXbeTitleID, "%c%c-%u", &titlechar[0], &titlechar[1], &titleno);
```

Comment:
This should just check if each component was scan'd correctly and return an error otherwise.

We shouldn't have to default to CX-9999, either the user provides a fully valid XBE Title ID or they don't.